tests/llmobs/test_llmobs_service.py

```diff
@@ -1403,20 +1403,7 @@ def test_activate_distributed_headers_activates_context(LLMObs, mock_logs):
             mock_activate.assert_called_once_with(dummy_context)
 
 
-def _task(llmobs_service, errors, original_pid, original_span_writer_id):
```

Comment:
i don't think this is used anywhere, cool if we remove? @Yun-Kim

Reply:
Yes, thanks for the catch